### PR TITLE
improve macos getting started

### PR DESCRIPTION
**Pull request by Anton-4**
using curl prevents the `roc can't be opened because Apple cannot check it...`

### CODE REVIEW

File: getting_started/README.md [hunk 1]
```diff
@@ -10,10 +10,10 @@ If you have a specific question, the [FAQ](../FAQ.md) might have an answer, alth
...
 ## Installation
 
-- [Linux x86_64](linux_x86_64.md)
-- [MacOS Apple Silicon](macos_apple_silicon.md)
-- [MacOS x86_64](macos_x86_64.md)
-- [Windows](windows.md)
+- [🐧 Linux x86_64](linux_x86_64.md)
+- [🍏 MacOS Apple Silicon](macos_apple_silicon.md)
+- [🍏 MacOS x86_64](macos_x86_64.md)
+- [🟦 Windows](windows.md)
```
Comment (Anton-4) [inline getting_started/README.md:16]: I added some unicode symbols here to catch the attention of the scanning reader.

**Review by ayazhafiz — APPROVED** [review]

File: getting_started/macos_x86_64.md [hunk 2]
```diff
@@ -7,12 +7,10 @@
 In order to develop in Roc, you need to install the Roc CLI,
 which includes the Roc compiler and some helpful utilities.
 
-1. Download the latest nightly from the assets [here](https://github.com/roc-lang/roc/releases).
-
-1. To prevent "roc can't be opened because Apple can't check it...":
+1. 1. Download the latest roc nightly using the terminal:
```
Comment (brian-carroll) [inline getting_started/macos_x86_64.md:10]: There's an extra `1.` here!
Seems like a mistake, as I checked "view file" and it looks a bit funny.

File: getting_started/macos_apple_silicon.md [hunk 3]
```diff
@@ -7,12 +7,10 @@
...
 which includes the Roc compiler and some helpful utilities.
 
-1. Download the latest nightly from the assets [here](https://github.com/roc-lang/roc/releases).
-
-1. To prevent "roc can't be opened because Apple can't check it...":
+1. Download the latest roc nightly using the terminal:
 
     ```sh
-    xattr -d com.apple.quarantine roc_nightly-macos_apple_silicon-latest.tar.gz
+    curl -OL roc_nightly-macos_apple_silicon-latest.tar.gz
```
Comment (joshuawarner32) [inline getting_started/macos_apple_silicon.md:13]: Doesn't curl need a full URL?

**Comment by Anton-4** [comment]
Thanks @brian-carroll and @joshuawarner32 :)
I've pushed another commit to fix those issues

**Review by rtfeldman — APPROVED** [review]
Nice, thanks @Anton-4! 🎉

**Pull request merged**